kore/src/Kore/Step/Rule.hs

```diff
@@ -66,13 +67,53 @@ import Kore.Attribute.Priority
 import Kore.Debug
 import Kore.Error
 import Kore.IndexedModule.IndexedModule
+import Kore.Internal.Alias
+    ( Alias (..)
+    )
 import Kore.Internal.ApplicationSorts
-import Kore.Internal.TermLike as TermLike
+import Kore.Internal.TermLike
```

Comment:
Why do you need all these imports? Why weren't they needed before? It seems your change does not concern them.